### ISSUE TITLE
Missing configure file

### ISSUE THREAD

**Issue by spkaluzny** [sramsay/nowa#2]
The current version of nowa is missing the file `configure`:
```
bolt:spk:370% pwd     
/home/spk/SRC/nowa-master

bolt:spk:371% ls
bootstrap.sh  CODE_OF_CONDUCT.md  doc      Makefile.am  src
ChangeLog.md  configure.ac        LICENSE  README.md

bolt:spk:369% ./configure
./configure: Command not found.
```

**Comment by sramsay** [comment]
The *latest release* definitely does contain the configure script (I just checked).

I suspect what you are doing is cloning the dev repository, which does not contain the configure script (because that script is automatically generated using autotools).

I would suggest [downloading the latest tagged release.](https://github.com/sramsay/nowa/releases/tag/2.0.3).  But if you'd rather work with the cloned repository, you can either run:

`autoreconf -i`

in the top directory (this assumes you have autoconf and automake installed on your system), or:

`./bootstrap.sh`

(which basically does the same thing).

**Comment by spkaluzny** [comment]
You are correct, I was using the dev repository. I did not notice you have tagged releases. Once I got the latest tagged release (and installed cJSON) all is working fine. Thanks.

**Comment by sramsay** [comment]
Excellent!  (I suppose I could make this clearer in the README . . . ;)
